bundle.Dockerfile

```diff
@@ -6,9 +6,9 @@ LABEL operators.operatorframework.io.bundle.metadata.v1=metadata/
 LABEL operators.operatorframework.io.bundle.package.v1=service-telemetry-operator
 LABEL operators.operatorframework.io.bundle.channels.v1=latest
 LABEL operators.operatorframework.io.bundle.channel.default.v1=latest
+LABEL operators.operatorframework.io.metrics.mediatype.v1=metrics+v1
```

Comment:
Don't ask me why this keeps bouncing around when running the bundle generator...